Sources/Extensions/Foundation/CalendarExtensions.swift

```diff
@@ -20,4 +20,23 @@ public extension Calendar {
     public func numberOfDaysInMonth(for date: Date) -> Int {
         return range(of: .day, in: .month, for: date)?.count ?? 0
     }
+    
+    /// SwifterSwift: Return the number of days in the year for specified 'Date'.
+    ///
+    /// - Parameter date: the date from which the year is obtained
+    /// - Returns: number of days in the year of 'date'
+    public func numberOfDaysInYear(for date: Date) -> Int {
```

Comment:
I'd say the same about this one... but I guess some years can have 366 days with leap year